hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -235,7 +239,28 @@ def setup_asset_locations(self, asset_locations: str) -> None:
                 assets["Object ID"] = range(1, len(assets.index) + 1)
 
         # Set the asset locations to the geometry variable (self.exposure_geoms)
+        # and set the geom name
         self.set_exposure_geoms(assets)
+        self.set_geom_names("buildings")
+
+    def set_geom_names(self, name: str) -> None:
```

Comment:
just for my understanding: the function set_geom_names is defined here but it is already used above.
is self.set_geom_names somewhere else defined beforehand?

Reply:
Good question - I did not even think about it because it just worked. It is indeed used in line 244 and only defined in line 246. I think that is possible because they are functions within a class and they are all loaded at initialization of the class. This page explains a bit more: https://stackoverflow.com/questions/29055698/does-the-order-of-variables-and-functions-matter-in-class-python . If you want to read about classes in Python, the documentation is here: https://docs.python.org/3/tutorial/classes.html, although I don't think it's the most clear documentation